src/lib.rs

```diff
@@ -56,6 +56,24 @@ where
     Err(())
 }
 
+#[cfg(target_env = "gnu")]
+mod glibc {
+    use std::path::Path;
+
+    // Attempt to compile a c program that links to strlcpy from the std
+    // library to determine whether glibc packages it.
+    pub(crate) fn has_strlcpy() -> bool {
```

Comment:
If this is intended to be a (developer-only) doc comment:

```suggestion
    /// Attempt to compile a c program that links to strlcpy from the [`std`]
    /// library to determine whether glibc packages it.
    pub(crate) fn has_strlcpy() -> bool {
```

Reply:
What's the benefit? It won't show up in the generated docs. Is a GUI code editor thing?